common/api-review/auth-exp.api.md

```diff
@@ -184,6 +185,16 @@ export const debugErrorMap: AuthErrorMap;
 // @public
 export function deleteUser(user: User): Promise<void>;
 
+// @public (undocumented)

```

Comment:
Needs documentation now that it's publicly exposed?

Reply:
Yes, we should do a sweep for all packages at some point and add any missing documentation. I will leave this PR as is for now.